packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/index.md

```diff
@@ -6,24 +6,26 @@ layout: Guides
 
 <ClassicDocOieVersionNotAvailable />
 
-You can connect your mobile app to Okta and sign users in by [opening a browser](/docs/guides/sign-into-mobile-app-redirect/), however, if you prefer that your users not leave your app, you need to build a custom sign-in UI with native controls and screens instead. Use this guide to build a customized sign-in experience inside your mobile application.
+You can connect your mobile app to Okta and sign users in by [opening a browser](/docs/guides/sign-into-mobile-app-redirect/). However, if you prefer that users not leave your app, build a custom sign-in UI with native controls and screens instead.
+
+Use this guide to build a customized sign-in experience inside your mobile application.
 
 <EmbeddedBrowserWarning />
 
-> **Note:** If the browser sign-in method works for your application, we recommend using that since building a custom sign-in UI takes more effort and development time.
+> **Note:** If the browser sign-in method works for your application, Okta recommends using that since building a custom sign-in UI takes more effort and development time.
```

Comment:
```suggestion
> **Note:** If the browser sign-in method works for your application, Okta recommends using that because building a custom sign-in UI takes more effort and development time.
```